### DISCUSSION TITLE
Add scikit-learn package to base environment

### DISCUSSION THREAD

**Discussion by fersebas** [PaddlePaddle/PaddleSpatial#113] [BaiduKDDCup2022] [answered]
Hi! Please if it is possible to add that package it would be useful for ours models. Thank you so much.

**Comment by ramber1836** [comment, marked as answer]
This package has already been supported in the base env.

**Reply by fersebas** [reply]
Where? I can not see this package here https://github.com/PaddlePaddle/PaddleSpatial/blob/main/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/preinstalled-pkgs/base_env_installed_packages.md

**Reply by ramber1836** [reply]
The list has already been updated. Please refer to the latest version.
